docs/language/aggregates/union.md

```diff
@@ -38,18 +38,19 @@ echo '1 2 3 3' | zq -z 'yield union(this)' -
 ```
 
 Mixed types create a union type for the set elements:
-```mdtest-command-issue-3610
```

Comment:
This fix was unique. Through black box testing I confirmed that the trailing `-issue-3610` was having the effect of _not_ running the example via `mdtest`. Furthermore, I looked at #3610 and saw it had been fixed, and indeed, I found that if I dropped that trailing `-issue-3610` and let mdtest do its thing, it runs clean (and doesn't run clean if I intentionally mucked the output). So it's now going to be mdtest-protected going forward. Please speak up if I that was the wrong decision, though!